docs/en/setup/backend/backend-storage.md

```diff
@@ -64,6 +65,16 @@ storage:
     segmentQueryMaxSize: ${SW_STORAGE_ES_QUERY_SEGMENT_SIZE:200}
 ```
 
+and there're also some configurations that are ES7 specific, as follows:
+
+```yaml
+storage:
+  # Index max result window, for segment deep pagination, usually we don't recommend to scroll too many pages,
+  # instead, give more query criteria (e.g. service id or time range), to narrow the query results.
+  # see https://www.elastic.co/guide/en/elasticsearch/guide/current/pagination.html for more information
+  indexMaxResultWindow: ${SW_STORAGE_ES_INDEX_MAX_RESULT_WINDOW:5000}
```

Comment:
You miss one level here, which could cause misguide. And should add comments in his YAML fragment to say, using other parameters above.